docs/payment-gateways/stripe.rst

```diff
@@ -0,0 +1,20 @@
+Stripe (supports Credit Cards)
+===========================================
+
+This gateway implements payments using `Stripe <https://stripe.com/>`_.
+
+.. table::
+
+    ================================== ======================================================================================================
+    Environment variable               Description
+    ================================== ======================================================================================================
+    ``STRIPE_PUBLIC_KEY``              Your Stripe's public key (test or live)
+    ``STRIPE_SECRET_KEY``              Your Stripe's secret key (test or live)
+    ``STRIPE_STORE_NAME``              Your store name to show in the checkout form
+    ``STRIPE_STORE_IMAGE``             An absolute or relative link of your store logo to show in the checkout form
+    ``STRIPE_PREFILL``                 Prefill the email adddress in the checkout form if set to ``True`` (default)
+    ``STRIPE_REMEMBER_ME``             Add "Remember Me" for future purchases in the checkout form if set to ``True`` (default)
+    ``STRIPE_LOCAL``                   Specify auto to display checkout form in the user's preferred language (default)
```

Comment:
I would suggest to do this, to be more straightforward:
```suggestion
    ``STRIPE_LOCAL``                   Specify ``auto`` to display checkout form in the user's preferred language (default)
```

Reply:
Thanks!